src/util/instanceStart.tsx

```diff
@@ -34,25 +33,27 @@ export const useInstanceStart = (instance: LxdInstance) => {
     instanceLoading.setLoading(instance, "Starting");
     const mutation =
       instance.status === "Frozen" ? unfreezeInstance : startInstance;
+
+    const instanceLink = (
+      <ResourceLink
+        type={instance.type}
+        value={instance.name}
+        to={`/ui/project/${instance.project}/instance/${instance.name}`}
+      />
```

Comment:
I think we should keep the old `<InstanceLink instance={instance} />` helper to avoid repeating this bit over and over again.

Reply:
Yeah agreed :+1: `InstanceLink` also gets used in other UI components, would you be fine if this PR introduced bits of non-notification related changes? Alternatively, I could create another component `InstanceLinkChip`, and for other PRs I will replace `InstanceLink` with it until all updates are done?

Reply:
Going with a new `InstanceLinkChip` component. I don't want to introduce non-notification related changes into this PR, and it would be simple enough to replace the components later.

Reply:
Yeah, you are right, we want to keep instance link for the instance table use case anyway long term as is. Better introduce a new one.

Reply:
While the helper was introduced, this bit is not using it yet. Along with a couple of other places.